src/System.Runtime/tests/Performance/Perf.String.cs

```diff
@@ -232,28 +233,104 @@ public void Substring_int_int(int size)
                         testString.Substring(startIndex, length);
         }
 
-        [Benchmark]
-        [MemberData(nameof(TestStringSizes))]
-        public void ToLower(int size)
+        [Flags]
+        public enum ChangeCaseOptions
         {
-            PerfUtils utils = new PerfUtils();
-            string testString = utils.CreateString(size);
+            None = 0,
+            UniqueString = 0x1,
+
+            MiddleTurkishI = 0x2,
+            MiddleDifferentCase = 0x4,
+            AllDifferentCase = 0x8,
+        }
+
+        public static IEnumerable<object[]> ChangeCaseMemberData() =>
+            from size in new[] { 1, 10, 500 }
+            from culture in new[] { "en-US" }
```

Comment:
Why parameterize culture but only ever use en-US? Is that the only culture we can rely on being installed?

Reply:
Mainly because I had other cultures in there, including tr-TR and invariant, and decided it was too many perf tests to check-in, but wanted to make it easy to still add them in locally.